controllers/applicationsController/getAllApplications.js

```diff
@@ -17,20 +16,14 @@ const getAllApplications = async (req, res, next) => {
   try {
     const { organizationId } = req.token;
     if (!mongoose.Types.ObjectId.isValid(organizationId)) {
-      return next(new CustomError(400, "Invalid OrganizationID"));
-    }
-
-    const organization = await Organizations.find({ _id: organizationId });
-    if (!organization) {
-      return next(new CustomError(400, "Invalid organization"));
+      return next(new CustomError(422, "Invalid OrganizationID"));
     }
```

Comment:
Why remove this?

Reply:
As explained in the first review conversation.